data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/SchemaDefinitionCacheTest.java

```diff
@@ -89,7 +90,7 @@ void subsequentCallsShouldGetTheSameSchemaObject(final SpecMilestone specMilesto
     // all the get methods should return the same object on subsequent calls,
     // as creation of the schema objects is relatively expensive
     methods.stream()
-        .filter(method -> method.getName().startsWith("get"))
+        .filter(method -> method.getAnnotation(NonSchema.class) == null)
```

Comment:
Won't this still include methods like hashCode and equals?

Reply:
good point but, nope. I think because we are listing methods from an `interface`